### ISSUE TITLE
Publish on gradle plugins?

### ISSUE THREAD

**Issue by simplesteph** [ChrisLMerrill/update4j-gradle-plugin#1]
Would be great to quickly try this out :)

**Comment by ChrisLMerrill** [comment]
Yeah...I haven't learned how to do that, yet. If more than a few people use this, then I'll give it a shot.

For now, the instructions at the end of the readme _should_ get you going.
